spec/unit/recipes/config_app_spec.rb

```diff
@@ -35,14 +33,17 @@
 
         it 'should render the init script for the default app' do
           if platform == 'centos' && version.to_i >= 7
-            expect(chef_run).to render_file('/usr/lib/systemd/system/myapp.service').with_content(/^PIDFile=\/var\/run\/wrapper.myapp.pid$/)
+            expect(chef_run).to render_file('/usr/lib/systemd/system/myapp.service'
+                                ).with_content(%r{^PIDFile=/var/run/wrapper.myapp.pid$})
           else
-            expect(chef_run).to render_file('/etc/init.d/myapp').with_content(/^APP_HOME=\/usr\/local\/yajsw_apps\/myapp$/)
+            expect(chef_run).to render_file('/etc/init.d/myapp'
+                                ).with_content(%r{^APP_HOME=/usr/local/yajsw_apps/myapp$})
           end
         end
 
         it 'should render the yajsw wrapper config for the default app' do
-          expect(chef_run).to render_file('/usr/local/yajsw_apps/myapp/conf/wrapper.conf').with_content(/wrapper\.java\.app\.jar=lib\/com\.company\.myapp\.jar$/)
+          expect(chef_run).to render_file('/usr/local/yajsw_apps/myapp/conf/wrapper.conf'
+                              ).with_content(%r{wrapper.java.app.jar=lib/com.company.myapp.jar$})
```

Comment:
In some places you line up `)` with the beginning of the method, and in other places you line it up with the matching `(`.  I'll never figure out you rubyists!